org.idempiere.p2.targetplatform/org.idempiere.p2.targetplatform.mirror.target

```diff
@@ -1,7 +1,400 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
 <?pde version="3.8"?>
-<target name="idempiere-12-local-mirror">
+<target includeMode="plugin" name="idempiere-12-local-mirror">
 	<locations>
-		<location path="${project_loc:org.idempiere.p2.targetplatform}/target/target-platform-repository" type="Directory"/>
+		<location path="${project_loc:org.idempiere.p2}/target/repository" type="Directory"/>
```

Comment:
this is for mirror, not target/repository and shouldn't have includeBundles either